Docker.DotNet.BasicAuth.nuspec

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Docker.DotNet.BasicAuth</id>
+    <version>2.124.3</version>
```

Comment:
Im actually confused here. What I was remembering was our other project [AppVeyor.yaml](https://github.com/Microsoft/Docker-PowerShell/blob/master/appveyor.yml) where you see we crack the `.json` in order to maintain a dev build. The one in the `.nuspec` is for the released version.

@swernli - Are we not doing dev builds for this AppVeyor here too?